### ISSUE TITLE
package names only parsed up to the first '.' (i.e. period char)

### ISSUE THREAD

**Issue by brent-ridian** [Nicolas-Reyland/findpydeps#3]
First, Nicolas, thank you very much for writing this software and making it available on PyPi.  I am grateful.

I just tried out findpydeps on one of my work repos, using a command line like this:

`findpydeps  -i .  --blocks  --functions  --follow-local-imports  --removal-policy 0  --strict  --no-header  > ./requirements_from_findpydeps.txt`

_I am confused about the results that findpydeps gave me._

In particular, my files have import lines involving [opentelemetry](https://opentelemetry.io/) like this:
```
from opentelemetry.exporter.cloud_trace import CloudTraceSpanExporter
from opentelemetry.instrumentation.logging import LoggingInstrumentor
from opentelemetry.propagate import set_global_textmap
from opentelemetry.propagators.cloud_trace_propagator import CloudTraceFormatPropagator
from opentelemetry.sdk.resources import Resource
from opentelemetry.sdk.trace import Tracer, TracerProvider
from opentelemetry.sdk.trace.export import BatchSpanProcessor, ConsoleSpanExporter
```

But findpydeps summarizes all those imports merely as
`opentelemetry`

In other words, instead of listing
```
opentelemetry.exporter.cloud_trace
opentelemetry.instrumentation.logging
etc
```
it seems as if findpydeps drops everything after the first '.' (i.e. period char).

**That is a bug, right?  I need the full package names to be listed, not just the first part.**

I note that with opentelemetry, even if findpydeps gave me the fully parsed names, like `opentelemetry.exporter.cloud_trace`, that that would only be half the battle.  For reasons that I cannot fathom, Google uses different names for the packages that you need to specify in your requirments.txt file.  For example, instead of listing something like
`opentelemetry-exporter-cloud-trace`
in your requirments.txt, you actually need to use
`opentelemetry-exporter-gcp-trace`

**Comment by Nicolas-Reyland** [comment]
Hello, thanks for giving me this feedback. Iirc, the behavior you describe as a bug is actually the intended behavior. I intended to group packages by _toplevel_ name. But it is possible to add an option to remove this step.

**Comment by Nicolas-Reyland** [comment]
Check out the `get_module_name_in_simple_import` function for this.

**Comment by Nicolas-Reyland** [comment]
There is now an option `--submodules-as-modules` that you will be able to use at the next release (which should be soon).

**Comment by Nicolas-Reyland** [comment]
The release `v0.2.6` implements this new option.
